src/nrniv/partrans.cpp

```diff
@@ -17,23 +17,40 @@
 #include <utility>
 #include <vector>
 
+#if NRNMPI
+#include "have2want.hpp"
+#endif
+
+
 #if NRNLONGSGID
 #if NRNMPI
-static void sgid_alltoallv(sgid_t* s, int* scnt, int* sdispl, sgid_t* r, int* rcnt, int* rdispl) {
-    if (nrn_sparse_partrans > 0) {
-        nrnmpi_long_alltoallv_sparse(s, scnt, sdispl, r, rcnt, rdispl);
-    } else {
-        nrnmpi_long_alltoallv(s, scnt, sdispl, r, rcnt, rdispl);
-    }
+static void sgid_alltoallv(Data<sgid_t>& s, Data<sgid_t>& r) if (nrn_sparse_partrans > 0) {
```

Comment:
```suggestion
static void sgid_alltoallv(Data<sgid_t>& s, Data<sgid_t>& r) { if (nrn_sparse_partrans > 0) {
```
i.e. `{` is missing (and need reformat)